pymc_marketing/clv/utils.py

```diff
@@ -597,3 +598,185 @@ def rfm_train_test_split(
     train_test_rfm_data["test_T"] = time_delta / time_scaler  # type: ignore
 
     return train_test_rfm_data
+
+
+def rfm_segments(
+    transactions: pd.DataFrame,
+    customer_id_col: str,
+    datetime_col: str,
+    monetary_value_col: str,
+    segment_config: dict | None = None,
+    observation_period_end: str | pd.Period | datetime | None | None = None,
+    datetime_format: str | None | None = None,
+    time_unit: str = "D",
+    time_scaler: float | None | None = 1,
+    sort_transactions: bool | None | None = True,
+) -> pd.DataFrame:
+    """
```

Comment:
I think this function will be very useful! Marketing practitioners love RFM segments. Hence, I suggest we add more context about the method (high level) in the docstring as in https://www.pymc-marketing.io/en/stable/api/generated/pymc_marketing.mmm.delayed_saturated_mmm.DelayedSaturatedMMM.html#pymc_marketing.mmm.delayed_saturated_mmm.DelayedSaturatedMMM (for example)